host-maipo.yaml

```diff
@@ -35,3 +35,8 @@ packages:
  - ceph-common glusterfs-fuse
  # OpenShift
  - origin-node origin-hyperkube origin-clients
+
+remove-from-packages:
+  # https://github.com/openshift/os/issues/275
+  - - lua
```

Comment:
This doesn't look like valid YAML syntax. Just verifying that it's proper for this implementation of the parser.

Reply:
http://yaml-online-parser.appspot.com/?yaml=remove-from-packages%3A%0A++%23+https%3A%2F%2Fgithub.com%2Fopenshift%2Fos%2Fissues%2F275%0A++-+-+lua%0A++++-+%2Fusr%2Fbin%2F.*&type=json

Reply:
(Also I did test this locally)

Reply:
Well alright :-)